### ISSUE TITLE
Debian shuts down when visiting certain sites like the Beagleboard Quick Start Guide PDF and others

### ISSUE THREAD

**Issue by RetroInVitro** [beagleboard/beagleboard-x15#7]
Looks like it shuts all the way down rather than just the OS crashing?  The 12V light is on but the on/off LED is off.

Video:  https://drive.google.com/open?id=0BxpH9rZh23d-YmlSU2hlWkFtcU0

Which logs should I take a look at?

**Comment by CreativeWolf** [comment]
Can you try saving the PDF file and open it using a PDF viewer, I'm trying to eliminate Chromium here.

As far as Chromium goes - see if you find a directory for Chromium in /home/user/.config/ 

Since the system is shutting down as well - check journalctl and see if you find anything there.

Update here and we can try to work this together.

PS: I've just received my X15 2 days ago, waiting to get a power supply, will give it a drive in next couple of days

**Comment by RetroInVitro** [comment]
I did an update and dist-upgrade and am no longer crashing on the BB X15 quick start guide.  I should note that it still crashes on other heavy content sites.  Offline, someone chimed in that they think it may have to do with the CPU maxing out and overclocking as each site that crash has the CPU pegged.  

If you have another easy to test way to eliminate Chromium, I'd appreciate it.  Thanks.

**Comment by CreativeWolf** [comment]
Ahh. A straightforward way would be to use another browser maybe?

How's the CPU usage if you are able to see it with some other sites - Yes, I watched the video again and you had htop open with CPU struggling there.

**Comment by RetroInVitro** [comment]
You're very right about just using another browser!

FWIW, I'm now unable to reproduce the shutdown after a standard update/upgrade.

**Comment by CreativeWolf** [comment]
That's interesting.
How's the CPU behaving with the maxing out situation? (Maybe you could try stressing it a little more than the usual)

**Comment by jadonk** [comment]
@RetroInVitro ping. Was this an over-heating issue?

**Comment by RetroInVitro** [comment]
I don't think it was.  After apt-get updated/upgraded, I had no more issues visiting the two sites with Chromium that shut it down.  Is there an internal cpu/gpu temperature sensor with values obtainable from CLI like "/opt/vc/bin/vcgencmd measure_temp" on other systems?
